html/sound.js

```diff
@@ -1,63 +1,67 @@
-var audio;
+let audio = {}
 
-function play(audioCoords, audioRot, audioSource){
-	
-	audio = new Howl({
+function play(audioCoords, audioRot, audioSource, uid){
+	audio[uid] = {}
+	audio[uid] = new Howl({
 		src: audioSource,
 		onplay:function(){
 			$.post('http://ultra-3dsound/onStart', JSON.stringify({
-				isSongPlaying: true})
+				isSongPlaying: true, uid: uid})
 		)},
 		onstop: function(){
 			$.post('http://ultra-3dsound/onEnd', JSON.stringify({
-				isSongPlaying: false})
+				isSongPlaying: false, uid: uid})
 		)},
 		onend: function(){
 			$.post('http://ultra-3dsound/onEnd', JSON.stringify({
-				isSongPlaying: false})
+				isSongPlaying: false, uid: uid})
 		)},
 		onloaderror: function(){
```

Comment:
`onloaderror` and `onplayerror` needs to post to `http://ultra-3dsound/onError` instead of `http://ultra-3dsound/onEnd`

Reply:
I will resolve this.